recipes/pourRNA/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "pourrna" %}
+{% set version = "1.0.1" %}
+{% set sha256 = "62635d3777403865a2801c78e42569abb232bc6a79abe34419ad4bf3c0701b0e" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ViennaRNA/pourRNA/releases/download/v{{ version }}/pourRNA-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - gengetopt
+    - viennarna
+  run:
+    - viennarna
+test:
+  commands:
+    - pourRNA --version
+
+about:
+  home: https://github.com/ViennaRNA/pourRNA/
+  license: GPL
+  license_file: COPYING
+  summary: "Compute local minima and energy barriers of a landscape."
+
```

Comment:
Is this tool published?
If so, you could add something like:
```
extra:
  identifiers:
- doi: xx/xxx.xx.xxx
```

Reply:
It is submitted. I will add it in the next version.